src/data/majors/anthr.ts

```diff
@@ -0,0 +1,108 @@
+import { CollegeOrMajorRequirement, Course } from '../../requirements/types';
+import {
+  ifCodeMatch,
+  courseMatchesCodeOptions,
+  includesWithSingleRequirement,
+  includesWithSubRequirements,
+  courseMeetsCreditMinimum,
+  courseIsFWS,
+} from '../../requirements/checkers';
+import { AdvisorGroup } from '../../tools/advisors/types';
+
+const anthrRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Sociocultural, Archaeological, and Biological Anthropology',
+    description:
+      'One course of 3 or more credits in each of the three subfields (sociocultural, archaeological, biological) from the list below.' +
```

Comment:
Do we need to check if these courses are >=3 credits?

Reply:
I mean they give you a fixed list of applicable courses, so I thought not. But might as well add in.

Reply:
Yeah so I looked around the other majors and never saw this being combined with a `minCredit` check. So gonna defer on this tell I get confirmation / cancellation.